packages/editor/src/components/rich-text/index.js

```diff
@@ -1020,7 +1025,28 @@ RichTextContainer.isEmpty = ( value ) => {
 };
 
 RichTextContainer.Content.defaultProps = {
-	format: 'rich-text',
+	format: 'string',
+	value: '',
+};
+
+RichTextContainer.getTextContent = ( html ) => {
```

Comment:
It feels like `RichText` component should be moved to `@wordpress/rich-text` package to avoid this duality where you can have two version of each of the public methods:
```js
import { join } from '@wordpress/rich-text';
import { RichText } from '@wordpress/editor';

join === RichText.join; // false
```

It feels like with the changes proposed `join` should remain private, and RichText.join would be the only public API.

Reply:
This has changed now.